AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/pyside_utils.py

```diff
@@ -739,7 +739,7 @@ async def async_wrapper():
         return run_async(result)
 
 
-async def open_context_menu(widget, pos=None, index=None, timeout=1.0):
+async def open_context_menu(widget, pos=None, index=None, timeout=5.0):
```

Comment:
This seems like an out of place change.  What's the purpose of this?  It might impact suites of tests.

Reply:
Project settings tool is opened by editor test `test_Menus_FileMenuOptions_Work`. After moving the android settings out of project.json that test failed because it opened the tool and it wasn't able to close it because it couldn't find android setting and it was asking the user to do an action, so the test eventually failed. That's why I did all the changes to make project settings tool read/write to android_project.json.

I also run all the editor tests and `test_BasicEditorWorkflows_LevelEntityComponentCRUD ` timed out due to this timer in `open_context_menu`, which is called from `trigger_context_menu_entry`, which is called from that test. I'm happy to take this change out of this PR, but I thought it was a nice change because if I run into it, then others can to.

My PC was running particularly slow yesterday, maybe that was factor, but still, it happened.

@amzn-phist @cgalvan with this info, what do you think? shall I revert this, keep it, or maybe other number of seconds?

Reply:
Thanks for the explanation!